study-builder/tenants/cmi/tenant.yaml

```diff
@@ -0,0 +1,29 @@
+rules:
+  - name: Register User in Pepper
+    script: ../register-user-in-pepper.js
```

Comment:
I put this put in the `tenants` directory instead of _inside_ the `cmi` directory because I think the rule is not specific to CMI. If we ever have another tenant in here, that tenant can point to the same rule.